modules/python/src2/gen2.py

```diff
@@ -27,14 +27,14 @@
 gen_template_call_constructor_prelude = Template("""new (&(self->v)) Ptr<$cname>(); // init Ptr with placement new
         if(self) """)
 
-gen_template_call_constructor = Template("""self->v.reset(new ${cname}${args})""")
+gen_template_call_constructor = Template("""self->v.reset(new ${cname}${py_args})""")
```

Comment:
Am I right that this PR will solve https://github.com/opencv/opencv/issues/17464?

Reply:
Exactly, but it can be improved by adding check that internal names doesn't match with names from user